ftw/file/tests/test_download.py

```diff
@@ -94,6 +94,26 @@ def test_inline_download(self, browser):
         self.assertEquals('inline; filename="file.doc"',
                           browser.headers['content-disposition'])
 
+    @browsing
+    def test_head_request(self, browser):
+        browser.default_driver = LIB_TRAVERSAL
+
+        browser.login().open(self.context, view='@@download', method='HEAD')
+
+        # We get a 302 but the testbrowser follows the redirect.
+        self.assertEqual(
+            'http://nohost/plone/f1/@@download/file/file.doc',
+            browser.url
+        )
+
+        # Let's open the url we got from the redirect.
+        browser.login().open(browser.url, method='HEAD')
```

Comment:
🤔 das ist eigentlich nicht notwendig - das ist bereits passiert (wegen automatischem Redirect).
Das Login ist auch doppelt.

Reply:
@jone, I have reworded the comment in the test to make my intentions more clear. What do you think?

Reply:
Fair 👍 Added comments.